api/v1beta1/horizontest_types.go

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// HorizonTestSpec defines the desired state of HorizonTest
+type HorizonTestSpec struct {
+	// +kubebuilder:validation:Required
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
+	// +kubebuilder:default="local-storage"
+	// StorageClass used to create PVCs that store the logs
+	StorageClass string `json:"storageClass"`
+
+	// AdminUsername is the username for the OpenStack admin user.
+	// +kubebuilder:validation:Required
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
```

Comment:
It seems like the `config/manifests/bases/test-operator.clusterserviceversion.yaml` was not updated. Can you please re-run `make bundle` locally on your machine and add the updated clusterserviceversion file into this PR, please?